src/main/java/usecase/alter_entry/AlterEntryResponseModel.java

```diff
@@ -0,0 +1,19 @@
+package usecase.alter_entry;
+
+import java.util.List;
+
+public class AlterEntryResponseModel {
```

Comment:
This use case should not have a response model because it doesn't really respond to something.

Remember that we use the `BillUpdateUseCase` to update the bill accordingly.

We may add some special feedback for altering an entry, but that would require more consideration and an issue.